### PR TITLE
feat(Communities): Present remotely destruct in progress in token holders list

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Presents state of remotely destruct in progress within token holders list.

Closes: #11775

### Affected areas
`SortableTokenHoldersList`

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

Screenshot from 2023-08-03 16-45-57

Note: The token was minted long time ago and for some reason total and remaining is presented as zero here. It's ok for new tokens.

[Kazam_screencast_00001.webm](https://github.com/status-im/status-desktop/assets/20650004/61f10430-6e9d-4022-a62e-72fd677a115c)

### CODE REVIEW

File: ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml [hunk 1]
```diff
@@ -181,14 +190,31 @@ StatusListView {
...
+                anchors.fill: parent
+                anchors.topMargin: delegate.topPadding
+
+                radius: Style.current.radius
+                color: "transparent"
+
+                SequentialAnimation on color {
+                    id: colorAnimation
+
+                    running: false
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml:210]: can't you set `running: model.remotelyDestructState === Constants.ContractTransactionStatus.InProgress`?

**Review by caybro — APPROVED** [review]
LGTM

one minor thing though

File: ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml (hunk 1 above)
Comment (micieslak) [reply ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml:210]: No, it doesn't work bc start needs to be triggered by change, not presence of given condition. In this case the animation would start immediately when burn is in progress, not when transits from InProgress to different state.

File: ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml [hunk 2]
```diff
@@ -181,14 +190,31 @@ StatusListView {
...
             }
 
+            Rectangle {
+                anchors.fill: parent
+                anchors.topMargin: delegate.topPadding
+
+                radius: Style.current.radius
+                color: "transparent"
+
+                SequentialAnimation on color {
```
Comment (alexjba) [inline ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml:207]: There is no spacing between rows

Screenshot 2023-08-03 at 22 52 31

**Review by alexjba — COMMENTED** [review]
LGTM! One minor issue found while testing. There is no spacing between rows.

Would be nice to have the Figma URL in storybook.

File: ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml (hunk 2 above)
Comment (micieslak) [reply ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml:209]: Thank you @alexjba for pointing that. Fixed!
And figma links added.

[Kazam_screencast_00002.webm](https://github.com/status-im/status-desktop/assets/20650004/9a2d9efe-cf34-4a18-8061-3567f626d5c1)

**Review by alexjba — APPROVED** [review]
Looking good! 👍

**Review by noeliaSD — APPROVED** [review]
LGTM! Tested!

**Pull request merged**